src/core/oauth2-authorize.js

```diff
@@ -52,10 +52,17 @@ export default function authorize ( { auth, authActions, errActions, configs, au
   }
   query.push("redirect_uri=" + encodeURIComponent(redirectUrl))
 
-  if (Array.isArray(scopes) && 0 < scopes.length) {
+  let scopesArray = []
+  if (Array.isArray(scopes)) {
+    scopesArray = scopes
+  } else if (typeof scopes !== "undefined" && typeof scopes.toArray === "function") {
+    scopesArray = scopes.toArray()
+  }
+
+  if (0 < scopesArray.length)  {
```

Comment:
legacy, but should use common conventions... `if (scopes.Array.length > 0) { do something }

Reply:
Sure, I will change this.